scripts/clang-format.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+if test x"$1" = x; then 
+	FILES=`git ls-files | grep '\.[ch]$'`
+fi
+if test x"$1" = xchanged; then 
+	FILES=`git diff --name-only | grep '\.[ch]$'`
```

Comment:
If the intention of `changed` is to affect files relevant to a new PR, wouldn't you also want to target untracked files?

Alternatively, will this collect files that you've staged for commit with `git add` but not yet committed?